src/System.Private.Xml/src/System/Xml/Xslt/XslCompiledTransform.cs

```diff
@@ -64,7 +64,7 @@ static XslCompiledTransform()
         private XmlWriterSettings _outputSettings = null;
         private QilExpression _qil = null;
 
-#if !uap
+#if !uap && !uapaot
```

Comment:
Just question: the reason to disable XSLT was RefEmit not available. But as serialization uses RefEmit on UAP, it means RefEmit works on UAP. If that's the case, do we still want to disable XSLT on UAP? Can we disable it on UAPAOT only?